Sources/Data/models/PlaylistFolder.swift

```diff
@@ -32,10 +35,20 @@ final public class PlaylistFolder: NSManagedObject, CRUD, Identifiable {
     let createdSort = NSSortDescriptor(key: "dateAdded", ascending: false)
     fetchRequest.sortDescriptors = [orderSort, createdSort]
 
-    if savedFolderContentsOnly {
+    if savedFolder {
       fetchRequest.predicate = NSPredicate(format: "uuid == %@", savedFolderUUID)
     } else {
-      fetchRequest.predicate = NSPredicate(format: "uuid != %@", savedFolderUUID)
+      if sharedFolders {
```

Comment:
nit: Instead of using the conditional use

        let folderIDPredicate = sharedFolders ? "sharedFolderId != nil" : "sharedFolderId == nil"
        fetchRequest.predicate = NSCompoundPredicate(andPredicateWithSubpredicates: [
          NSPredicate(format: "uuid != %@", savedFolderUUID),
          NSPredicate(format: folderIDPredicate)
        ])